src/itemlist_formaction.cpp

```diff
@@ -86,8 +86,10 @@ void itemlist_formaction::process_operation(operation op, bool automatic, std::v
 			if (itempos < visible_items.size()) {
 
 				bool browser_success = v->open_in_browser(visible_items[itempos].first->link());
-				if (!browser_success)
+				if (!browser_success) {
+					v->show_error(_("browser failed to open the link"));
```

Comment:
I'm sorry to cavil on such a minor detail, but it's user-facing and thus quite important: can you please make this message a whole sentence, with capital "b" at the beginning and a period at the end? All the other call to `show_error` do that.